src/styles/blog.css

```diff
@@ -72,18 +72,20 @@
 }
 
 .blog-card__image-container {
-  display: flex;
-  justify-content: center;
-  align-items: center;
   overflow: hidden;
   width: 360px;
-  height: 160px;
+  padding-top: 52.5%;
   border-radius: 4px;
+  position: relative;
 }
 
 .blog-card__image {
-  min-height: 100%;
   width: 100%;
```

Comment:
lets have the image `object-fit` mode set to `cover`. This way the images will all have the same render size and we can fix the ones that zoom in over time

Reply:
updated

Reply:
The aspect ratio on mobile is wrong

Reply:
updated